styles/main.css

```diff
@@ -255,34 +261,48 @@ header nav .burger {
         display: block;
     }
     header nav ul {
-        display: block;
-        direction: ltr;
-        padding-top: 1em;
-        padding-bottom: 1em;
+        margin-top: 1em;
+        margin-bottom: 1em;
         text-align: center;
+        flex-direction: column;
     }
-    header nav ul li {
-        transform: translate(-5%);
+    header nav ul li a {
+        border: none;
     }
-    header nav ul li:hover {
+    header nav ul li a:hover {
         text-decoration: underline;
         border: none;
     }
     header nav .burger,
     header nav .burger::before,
     header nav .burger::after {
-        content: "";
         display: block;
+        width: 40px;
         margin: 0 auto;
         border-width: 5px;
         border-top-style: solid;
         border-color: lightcoral;
-        margin: 5px 0;
         cursor: pointer;
     }
+    header nav .burger::before,
+    header nav .burger::after {
+        content: "";
+        position: absolute;
+    }
+    header nav .burger::before {
+        top: 5px;
+    }
+    header nav .burger::after {
+        top: 15px;
+    }
     header nav .burger {
         margin: 1em auto;
-        width: 40px;
         height: 25px;
+        position: relative;

```

Comment:
The main burger element needs to have a relative position so that the pseudo elements can be place relative to it